docs/for-developers/.pages

```diff
@@ -0,0 +1,3 @@
+title: For developers
```

Comment:
I'm not sure we'd benefit from a new page that only points to the GitHub... What about keeping the single page, having "For researchers" as usual, and "For developers" at the end with a one-liner pointing to GitHub?

Reply:
I don't think "Deploying locally" is a proper page to direct developers that want to contribute to REANA. It is hidden in a not obvious place.